demoandroid/src/main/java/com/andretietz/retroauth/demo/LoginActivity.kt

```diff
@@ -40,7 +40,6 @@ class LoginActivity : AuthenticationActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_login)
-    Timber.plant(Timber.DebugTree())
```

Comment:
A `DebugTree` was already planted in `MainActivity`, which lead to duplicated logs.